custom-login/package.json

```diff
@@ -11,48 +11,24 @@
     "pretest": "npm run pretest --prefix ../"
   },
   "dependencies": {
-    "@okta/okta-auth-js": "^4.3.0",
-    "@okta/okta-signin-widget": "^5.1.0",
-    "@okta/okta-vue": "^3.0.0",
-    "axios": "^0.20.0",
+    "@okta/okta-auth-js": "^4.6.1",
+    "@okta/okta-signin-widget": "^5.2.3",
+    "@okta/okta-vue": "git://github.com/okta/okta-vue.git#sw-support-vue-3-OKTA-353048",
```

Comment:
need to be updated to 4.0 or dev-4.0 branch when the PR in okta-vue get merged (before okta-vue@4.0 release)